eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -147,81 +147,8 @@ stages:
               Cloud: ${{ cloud.key }}
 
 - ${{ if contains(variables['Build.DefinitionName'], 'tests-weekly')  }}:
```

Comment:
Do you want to also include this condition in the tests.yml or in the template? Otherwise I think this job will run every day and not just weekly.

Reply:
@weshaggard you're right, there's really no value in running this every day. I'll add the condition to the template in another PR.